Sources/KituraContracts/Contracts.swift

```diff
@@ -35,6 +37,13 @@
 public struct RequestError: RawRepresentable, Equatable, Hashable, Comparable, Error, CustomStringConvertible {
     public typealias RawValue = Int
 
+    /// Representation of the error body
+    /// May be a type-erased Codable object or a Data (in a particular format)
+    public enum ErrorBody {
```

Comment:
Did you consider using the `struct` approach again, just in case we ever want a third option?

Reply:
I thought about it, but I'm not sure it's necessary here. Unlike the `BodyFormat` case, the function seems fairly tightly defined and fitted to these two cases.